src/lib/viewers/image/ImageBaseViewer.js

```diff
@@ -434,6 +435,42 @@ class ImageBaseViewer extends BaseViewer {
         }
     }
 
+    /**
+     * Prints image using an an iframe.
+     *
+     * @return {void}
+     */
+    print() {
+        const browserName = Browser.getName();
+
+        /**
+         * Called async to ensure resource is loaded for print preview. Then removes listener to prevent
+         * multiple handlers.
+         *
+         * @return {void}
+         */
+        const defaultPrintHandler = () => {
+            if (browserName === BROWSER_INTERNET_EXPLORER || browserName === BROWSER_EDGE) {
```

Comment:
It looks like this logic is duplicated line-for-line in PlainTextViewer. Not sure it's worth refactoring, though, since the viewers aren't closely related.

Reply:
I can follow up on that sometime as a cleanup task.